index.html

```diff
@@ -114,7 +114,80 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
-		
+		 <!-- ================================================ -->
+        <!-- ==================  TEMPLATE  ================== -->
+        <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+ <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+ <div class="card">
+  <p class="name">Raghav</p>
+  <p class="contact">
+    <i class="fab fa-twitter"></i>
+    <a href="https://twitter.com/raghavu37260366" target="_blank">raghavu37260366/a>
+  </p>
+  <p class="about">hey there!, i'm raghav and i like reading books usually biographies.</p>
+  <div class="resources">
+    <p>3 Useful Dev Resources</p>
+    <ul>
+      <li>
+        <a href="https://youtube.com/playlist?list=PLu0W_9lII9agiCUZYRsvtGTXdxkzPyItg" target="_blank" title="first resource">codwwithharry web development course</a>
+      </li>
+      <li>
+        <a href="https://youtube.com/playlist?list=PLfqMhTWNBTe3H6c9OGXb5_6wcc1Mca52n" target="_blank" title="Second Resource">aman dhattarwal web development course</a>
+      </li>
+      <li>
+        <a href="https://youtube.com/playlist?list=PL-Jc9J83PIiEeD3I4VXETPDmzJ3Z1rWb4" target="_blank" title="third resource">pepcoding web development course</a>
+      </li>
+    </ul>
+  </div>
+</div>
+<!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
change end template to your name. and put your card below the official template

Reply:
thanks alot